terraform/Mo/rancher-master/rancher-master.tf

```diff
@@ -24,7 +23,9 @@ resource "openstack_compute_instance_v2" "rancherMaster" {
     name = "${module.region.regional_public_network_name}"
   }
 
-  depends_on = ["openstack_blockstorage_volume_v1.rancherMaster_root_volume"]
+  depends_on = ["openstack_blockstorage_volume_v1.rancherMaster_root_volume"]  
```

Comment:
depends_on is a list, so repeating the command does overwrite the previous command. you need to have it in the same list